src/common/components/hosts/MassChangeHostnameModal.tsx

```diff
@@ -254,11 +257,11 @@ const MassChangeHostnameModal: React.FC<MassChangeHostnameModalProps> = ({
   const [patchingHost, setPatchingHost] = React.useState<number>(0);
 
   const selectedHosts = hosts.filter((h) => selectedHostIDs.includes(h.id));
-
+  const { t } = useTranslation();
   return (
     <Modal
-      aria-label="Change hostnames dialog"
-      title="Change hostnames"
+      aria-label="Change hostname dialog"
```

Comment:
Why removed the plural here? This is for the mass actions.

Reply:
Is a typo.